packages/xml/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "outDir": "dist",
     "rootDir": ".",
     "tsBuildInfoFile": "temp/tsconfig.tsbuildinfo",
+    "verbatimModuleSyntax": true,
```

Comment:
Xml library was written with this one but due to the dec generation had to disable